Riot/Modules/Settings/SettingsViewController.m

```diff
@@ -3257,25 +3374,26 @@ - (void)toggleEnableRingingForGroupCalls:(UISwitch *)sender
 
 - (void)toggleEnableThreads:(UISwitch *)sender
```

Comment:
you could comment the whole method ?

Reply:
If I do this, I also need to comment the line with the selector call, line 2581 `[labelAndSwitchCell.mxkSwitch addTarget:self action:@selector(toggleEnableThreads:) forControlEvents:UIControlEventTouchUpInside];` 
So I think it's better to avoid commenting this.